exercise4/playbook.yml

```diff
@@ -0,0 +1,71 @@
+- name: run webservers
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+
+    - name: trello_api_key
+      prompt: What is your trello api key?
+      private: false
```

Comment:
Fine here (and actively helpful while testing!) but even though it makes no difference for our current use case, I'd defensively suggest we switch these back to private on a real project, to prepare for potential changes, e.g. if someone lifted this playbook to run in a pipeline, we wouldn't want them to have to think about this to avoid potentially leaking secrets into the logs there